README-Mandrel.md

```diff
@@ -0,0 +1,45 @@
+# Mandrel
+
+Mandrel is a downstream distribution of the GraalVM community edition.
+Mandrel's main goal is to provide a `native-image` in-place replacement to be used by [Quarkus](https://quarkus.io).
+
+## How Does Mandrel Differ From Graal
+
+Mandrel releases may differ from the corresponding Graal releases in the following ways:
+  * Mandrel releases may include backports from Graal master that the corresponding Graal release doesn't include
+  * Mandrel may include features and/or fixes that are not yet available on Graal upstream
+  * Mandrel is built using upstream OpenJDK, while Graal is built using labs-OpenJDK
```

Comment:
@maxandersen @adinn please review ^^

Reply:
How about this:

Mandrel releases are built from a code base derived from the upstream GraalVM code base, with only very minor changes but some significant exclusions. They support the same native image capability as GraalVM with no significant changes to functionality. They do not include support for Polyglot programming via the Truffle interpreter and compiler framework. In consequence, it is not possible to extend Mandrel by downloading languages from the Truffle language catalogue.

Mandrel is also built slightly differently to GraalVM, using the standard OpenJDK project release of jdk11u. This means it does not profit from a few small enhancements that Oracle have added to version of OpenJDK used to build their own GraalVM downloads. Most of these enhancements are to the JVMCI module that allows the Graal compiler to be run inside OpenJDK. The others are small cosmetic changes to behaviour. These enhancements may in some cases cause minor differences in the progress of native image generation. They should not cause the resulting images themselves to execute in a noticeably different manner.

Reply:
Integrated ^^